open-metadata-implementation/adapters/open-connectors/governance-daemon-connectors/open-lineage-connectors/open-lineage-janus-connector/src/main/java/org/odpi/openmetadata/openconnectors/governancedaemonconnectors/openlineageconnectors/janusconnector/buffergraph/BufferGraphConnector.java

```diff
@@ -34,36 +35,23 @@
     private static final Logger log = LoggerFactory.getLogger(BufferGraphConnector.class);
     private JanusGraph bufferGraph;
     private GraphVertexMapper graphVertexMapper = new GraphVertexMapper();
-
     private JanusGraph mainGraph;
 
 
-    /**
-     * Initialize the connector.
-     *
-     * @param connectorInstanceId  - unique id for the connector instance - useful for messages etc
-     * @param connectionProperties - POJO for the configuration used to create the connector.
-     */
-    @Override
-    public void initialize(String connectorInstanceId, ConnectionProperties connectionProperties) {
-        super.initialize(connectorInstanceId, connectionProperties);
-        initializeGraphDB();
-    }
-
-    /**
-     * Indicates that the connector is completely configured and can begin processing.
-     *
-     * @throws ConnectorCheckedException there is a problem within the connector.
-     */
-    public void start() throws ConnectorCheckedException
-    {
-        super.start();
-    }
-
-    private void initializeGraphDB(){
+    public void initializeGraphDB() throws OpenLineageException {
         String graphDB = connectionProperties.getConfigurationProperties().get("graphDB").toString();
         GraphFactory graphFactory = new GraphFactory();
-        this.bufferGraph = graphFactory.openGraph(graphDB,connectionProperties);
+        try {
+            this.bufferGraph = graphFactory.openGraph(graphDB, connectionProperties);
+        } catch (JanusConnectorException error) {
+            throw new OpenLineageException(500,
```

Comment:
JanusConnectorException suppose to be logged as error before throwing another exception.
Otherwise the initial exception is lost and debugging and diagnose is very difficult.